lightly_solution.ipynb

```diff
@@ -11,318 +11,108 @@
+      "├── \u001b[01;34mshoes\u001b[00m [26 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mshorts\u001b[00m [25 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mskirt\u001b[00m\n",
+      "│   ├── \u001b[01;35m2459a9ab-b654-4aef-be7c-89340da78cec.jpg\u001b[00m\n",
+      "│   ├── ...\n",
+      "│   └── \u001b[01;35mecdc68a9-12c7-449b-ab3a-4b2a8e71b197.jpg\u001b[00m\n",
+      "└── \u001b[01;34mt-shirt\u001b[00m [81 entries exceeds filelimit, not opening dir]\n",
+      "\n",
+      "10 directories, 14 files\n"
      ]
     }
    ],
    "source": [
-    "\"\"\" Register the Lightly Worker. \"\"\"\n",
-    "\n",
-    "from lightly.api import ApiWorkflowClient\n",
-    "\n",
-    "client = ApiWorkflowClient(token=lightly_token)\n",
-    "\n",
-    "# Create a Lightly Worker. If a worker with this name already exists, the id of the existing\n",
-    "# worker is returned.\n",
-    "worker_id = client.register_compute_worker(name=\"clothing-worker\")\n",
-    "\n",
-    "os.environ[\"LIGHTLY_WORKER_ID\"] = worker_id\n",
-    "!echo \"LIGHTLY_WORKER_ID =\" $LIGHTLY_WORKER_ID"
+    "# Set the path to the dataset.\n",
+    "# Here we downlaod and use a dataset containing 354 images showing clothes.\n",
+    "# If you want to use your own dataset, just set the dataset_path to it.\n",
+    "dataset_path = \"./dataset_clothing_images\"\n",
+    "!git clone https://github.com/lightly-ai/dataset_clothing_images.git {dataset_path}\n",
+    "!tree --filelimit=12 {str(dataset_path)}"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
-    "\"\"\" Create a dataset in the Lightly platform and configure the datasource. \"\"\"\n",
-    "\n",
+    "# Schedule a run\n",
     "from lightly.api import ApiWorkflowClient\n",
-    "from lightly.openapi_generated.swagger_client import DatasetType\n",
-    "from lightly.openapi_generated.swagger_client import DatasourcePurpose\n",
+    "from lightly.openapi_generated.swagger_client import DatasetType, DatasourcePurpose\n",
+    "\n",
+    "lightly_token = \"CHANGE_ME\" # Copy the API_TOKEN from https://app.lightly.ai/preferences.\n",
     "\n",
     "# Create the Lightly client to connect to the API.\n",
     "client = ApiWorkflowClient(token=lightly_token)\n",
     "\n",
+    "# Register the Lightly Worker.\n",
+    "worker_id = client.register_compute_worker(name=\"clothing-worker\")\n",
+    "\n",
     "# Create the dataset on the Lightly Platform.\n",
-    "try:\n",
-    "    client.create_dataset(\n",
-    "        dataset_name=\"clothing-small\",\n",
-    "        dataset_type=DatasetType.IMAGES\n",
-    "    )\n",
-    "except:\n",
-    "    client.set_dataset_id_by_name(\"clothing-small\")\n",
+    "client.create_dataset(\n",
+    "    dataset_name=\"clothing-small\",\n",
+    "    dataset_type=DatasetType.IMAGES,\n",
+    ")\n",
```

Comment:
Should we add a comment above that says: "You might have to change the name if a dataset with the same name already exists." ?

Reply:
I can't find this comment in the file.

Reply:
Somehow it got lost again, added it again.